src/pages/ImportWorkflow/ImportWorkflow.js

```diff
@@ -85,9 +86,15 @@ export const ImportWorkflow = ({ path, version, source }) => {
         method_output_mappings: [],
       };
       const res = await Ajax(signal).Cbas.methods.post(appUrls.cbasUrl, postRequestBody);
+      const methodId = res.method_id;
 
       await setAzureCookieOnUrl(signal, appUrls.cbasUiUrl, true);
-      window.location = `${appUrls.cbasUiUrl}#submission-config/${res.method_id}`;
+      // TODO: Remove this logic when fully migrated into Terra-UI and Cbas UI has been deprecated
+      if (isAzure) {
```

Comment:
I think instead of `isAzure` being passed in the function, we should be directly able to access the config flag inside the if condition like
```
if(isAzureWorkflowsTabVisible()) {
...
}
```